### DISCUSSION TITLE
I might be dumb but how do you select what enemies can and cannot mimic voices?

### DISCUSSION THREAD

**Discussion by LOVETRAIN01** [qwbarch/mirage#97] [Q&A] [answered]
I know masked enemies mimic by default, and others don't, and I looked through the config and couldn't find where to change this, the mod desc said there was a way, but Mirage.enemies.cfg is blank and idk where else to look

**Comment by qwbarch** [comment, marked as answer]
Maybe ``Mirage.Enemies.cfg`` didn't generate properly. Try hosting a game and then try again. ``Mirage.Enemies.cfg`` gets generated when you load into the game (when you get into the ship, not just the main menu), which I probably should've specified in the README

**Reply by LOVETRAIN01** [reply]
This was exactly it! I wasn't expecting it to individually detect each modded enemy and allow me to pick even among them who gets to mimic and who doesn't! Insanely cool feature, thank you for your help!
